pyanaconda/payload/dnfpayload.py

```diff
@@ -1207,6 +1216,47 @@ def updateBaseRepo(self, fallback=True, checkmount=True):
                 if repo in enabled_repos:
                     self._fetch_md(repo)
 
+    def _getBaseRepoUrl(self, url):
+        """Try to find base repository from the treeinfo file."""
+        if self._treeinfo:
+            variants = self._treeinfo.variants
+
+            for variant in variants:
+                if variant in self.DEFAULT_REPOS:
+                    variant_obj = variants[variant]
+                    log.info("Found base repository in treeinfo file")
+                    if variant_obj.paths.repository == ".":
+                        log.debug("Treeinfo points base repository to installation tree root")
+                    else:
+                        url = url + "/" + variant_obj.paths.repository
+                        log.debug("Treeinfo points base repository to %s", url)
```

Comment:
I don't understand the messages.

Reply:
As I explained above, you can point to a repository or an repository root.
In this case the treeinfo path of the repository is `.` -- in other words it is repository root.